src/pages/thank-you.js

```diff
@@ -0,0 +1,20 @@
+import React  from 'react';
+import Footer from '../components/footer.js';
+import Header from '../components/header.js';
+
+var ThankYou = React.createClass({
```

Comment:
Possibly. I'm not super worried about the pages, components, or html yet. Same goes for less. It's easy to modify that once I get started.